cmd/commento/main.go

```diff
@@ -15,23 +16,23 @@ func main() {
 	if err != nil {
 		Die(err)
 	}
-	
+
+	// Load configuration from the environment.
+	// Values in earlier files will take precedence over later values
+	//    Ex. A COMMENTO_PORT value in .env.development.local will be used
+	//        even if COMMENTO_PORT is exists in a .env.development file
+	for _, envFile := range []string{".env.development.local", ".env.test.local", ".env.production.local", ".env.local", ".env.development", ".env.test", ".env.production", ".env"} {
+		godotenv.Load(envFile)
+	}
+
 	fs := http.FileServer(http.Dir("assets"))
 	
 	http.Handle("/assets/", http.StripPrefix("/assets/", fs))
 	http.HandleFunc("/", IndexHandler)
 	http.HandleFunc("/create", CreateCommentHandler)
 	http.HandleFunc("/get", GetCommentsHandler)
 	
-	var port string
-	
-	if fromEnv := os.Getenv("PORT"); fromEnv != "" {
-		port = ":" + fromEnv
-	} else {
-		port = ":8080"
-	}
-	
-	if demoEnv := os.Getenv("DEMO"); demoEnv == "true" {
+	if demoEnv := os.Getenv("COMMENTO_DEMO"); demoEnv == "true" {
```

Comment:
That's not bad. What if the user enters 'truee' or 'fals' or has some other writing error? We should probably check that a COMMENTO_DEMO actually parses to a boolean:

```
if s, err := strconv.ParseBool(os.Getenv("COMMENTO_DEMO")); err != nil {
		Logger.Infof("COMMENTO_DEMO not set properly, defaulting to no demo")
}
```